finder-app/manual-linux.sh

```diff
@@ -48,33 +68,100 @@ then
 fi
 
 # TODO: Create necessary base directories
+# Creating root file system in output directory and making basic directories
+mkdir "$OUTDIR/rootfs"
+cd "$OUTDIR/rootfs"
+mkdir -p bin dev etc home lib lib64 proc sbin sys tmp usr var
+mkdir -p usr/bin usr/bin usr/sbin
+mkdir -p var/log
+
 
 cd "$OUTDIR"
 if [ ! -d "${OUTDIR}/busybox" ]
 then
-git clone git://busybox.net/busybox.git
-    cd busybox
+git clone https://github.com/mirror/busybox.git
+#git clone git@github.com:mirror/busybox.git
+
+	    cd busybox
     git checkout ${BUSYBOX_VERSION}
     # TODO:  Configure busybox
+    echo "Busy box configuration"
+    make distclean
+    make defconfig
+   
 else
+    echo "Busy box already configured"
     cd busybox
 fi
 
 # TODO: Make and install busybox
+make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE}
+make CONFIG_PREFIX="${OUTDIR}/rootfs" ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} install
+
+cd "${OUTDIR}"
 
 echo "Library dependencies"
-${CROSS_COMPILE}readelf -a bin/busybox | grep "program interpreter"
-${CROSS_COMPILE}readelf -a bin/busybox | grep "Shared library"
+${CROSS_COMPILE}readelf -a ./rootfs/bin/busybox | grep "program interpreter"
+${CROSS_COMPILE}readelf -a ./rootfs/bin/busybox | grep "Shared library"
+
+cp /home/bhakti/Downloads/arm-gnu-toolchain-13.3.rel1-x86_64-aarch64-none-linux-gnu/aarch64-none-linux-gnu/libc/lib/ld-linux-aarch64.so.1 ${OUTDIR}/rootfs/lib
+cp /home/bhakti/Downloads/arm-gnu-toolchain-13.3.rel1-x86_64-aarch64-none-linux-gnu/aarch64-none-linux-gnu/libc/lib64/libm.so.6 ${OUTDIR}/rootfs/lib64/
+cp /home/bhakti/Downloads/arm-gnu-toolchain-13.3.rel1-x86_64-aarch64-none-linux-gnu/aarch64-none-linux-gnu/libc/lib64/libresolv.so.2 ${OUTDIR}/rootfs/lib64/
+cp /home/bhakti/Downloads/arm-gnu-toolchain-13.3.rel1-x86_64-aarch64-none-linux-gnu/aarch64-none-linux-gnu/libc/lib64/libc.so.6 ${OUTDIR}/rootfs/lib64/
+
+#TODO: Add library dependencies to rootfs
+
+echo "Library Dependencies added"
+#echo $(ls ${OUTDIR}/rootfs/lib64)
+#echo $(ls ${OUTDIR}/rootfs/lib)
 
-# TODO: Add library dependencies to rootfs
 
 # TODO: Make device nodes
+# creating a device/special file - giving permision as mode - permissions - path for newly created file - character file - major no(memory driver) - minor no(specific driver)
+#cd "${OUTDIR}/rootfs"
+
+sudo rm -f /dev/null
+sudo mknod -m 666 /dev/null c 1 3
+
+sudo rm -f /dev/console
+sudo mknod -m 600 /dev/console c 5 1
+
+echo "Noed Added"
 
 # TODO: Clean and build the writer utility
+# Go to finder app directory
+cd /home/bhakti/Work/assignments-3-and-later-BhaktiRamani/finder-app
```

Comment:
@BhaktiRamani don't hard-code your finder-app path, use `FINDER_APP_DIR` which is setup for you based on the location of this script file.